client/onboarding/strings.tsx

```diff
@@ -187,6 +187,10 @@ export default {
 		from_3_to_6months: __( '3 – 6 months', 'woocommerce-payments' ),
 		more_than_6months: __( '6+ months', 'woocommerce-payments' ),
 	},
+	restoredState: __(
+		'We have restored your previous session. You can pick up where you left off or go back to a previous step to make changes. ',
```

Comment:
Nitpick: what do you think about adding a comma to this sentence:

```You can pick up where you left off, or go back to a previous step to make changes.```

Even though it's grammatically correct either way, I think the comma helps present the options as two distinct choices (go back or continue). 🙂 Happy to go with whatever you think is best!

Reply:
Actually I added the comma on an earlier version, not sure why I removed it later 🤔 Thanks for catching it! _Addressed in 738f73b_